src/promptflow/promptflow/_sdk/_submitter/utils.py

```diff
@@ -252,14 +252,13 @@ def init_env(cls, environment_variables):
 
     @staticmethod
     def resolve_connections(
-        flow: Flow, client=None, *, connections_to_ignore=None, connections_to_add: List[str] = None
+        flow: Flow,
+        client=None,
+        *,
+        connections_to_ignore=None,
+        connections_to_add: List[str] = None,
+        environment_variables_overrides: Dict[str, str] = None,
     ) -> dict:
-        from promptflow._sdk.entities._flow import FlexFlow
-
-        if isinstance(flow, FlexFlow):
-            # TODO(2898247): support prompt flow management connection for eager flow
```

Comment:
this is now supported?

Reply:
Yes, flex flow support prompt flow management connection by environment variable connection